browser/brave_stats_updater.cc

```diff
@@ -79,7 +79,8 @@ void BraveStatsUpdater::Start() {
   DCHECK(!server_ping_startup_timer_);
   server_ping_startup_timer_ = std::make_unique<base::OneShotTimer>();
 #if BUILDFLAG(ENABLE_BRAVE_REFERRALS)
-  if (pref_service_->GetBoolean(kReferralInitialization)) {
+  if (pref_service_->GetBoolean(kReferralInitialization) ||
+        pref_service_->GetBoolean(kReferralCheckedForPromoCodeFile)) {
```

Comment:
@simonhong does this look correct? @keur was mentioning you had found this in the original PR

Reply:
https://github.com/brave/brave-core/pull/5144#discussion_r402772745

Reply:
Yes, this change seems correct because `kReferralInitialization` is only set to true for new installs.
If `kReferralCheckedForPromoCodeFile` is set, `kReferralInitialization` is remained as false.
As description stated, all installs before `kReferralInitialization` is introduce, we should check `kReferralCheckedForPromoCodeFile`.